docs/api.rst

```diff
@@ -485,6 +485,55 @@ Possible Responses
 | 500 Server Error | Internal Raiden node error|
 +------------------+---------------------------+
 
+
+Querying token specific events
```

Comment:
This section can be misleading, the end-point does not return token specific events but ChannelManager/token network events